Riot/Modules/Rooms/ShowDirectory/ShowDirectoryViewController.swift

```diff
@@ -18,18 +18,18 @@
 
 import UIKit
 
-@objc protocol SearchableDirectoryViewControllerDelegate: NSObjectProtocol {
-    func searchableDirectoryViewControllerDidCancel(_ viewController: SearchableDirectoryViewController)
-    func searchableDirectoryViewControllerDidSelect(_ viewController: SearchableDirectoryViewController, room: MXPublicRoom)
-    func searchableDirectoryViewControllerDidTapCreateNewRoom(_ viewController: SearchableDirectoryViewController)
-}
-
-final class SearchableDirectoryViewController: MXKViewController {
+final class ShowDirectoryViewController: UIViewController {
+    
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let aConstant: Int = 666
```

Comment:
Sorry for this useless constant in the template :)

Reply:
Done